purekit/src/main/kotlin/com/virgilsecurity/purekit/pure/client/HttpKmsClient.java

```diff
@@ -89,6 +89,7 @@ public HttpKmsClient(String appToken, String serviceAddress) {
         return client.firePost(
                 request,
                 AvailableRequests.DECRYPT_REQUEST.getType(),
+                //TODO no need to path headers because of value by default
```

Comment:
AFAIK Java doesn't support defaults values, even when calling Kotlin functions.

Reply:
There is `firePost` signature without `headers` parameters. Just inspect it in IDE